worker/migrationminion/worker.go

```diff
@@ -149,7 +172,25 @@ func (w *Worker) doQUIESCE(status watcher.MigrationStatus) error {
 }
 
 func (w *Worker) doVALIDATION(status watcher.MigrationStatus) error {
-	err := w.validate(status)
+	attempt := retry.StartWithCancel(
+		retry.LimitCount(maxRetries, retry.Exponential{
+			Initial: initialRetryDelay,
+			Factor:  retryBackoffFactor,
+			Jitter:  true,
+		}),
+		w.config.Clock,
+		w.catacomb.Dying(),
+	)
+	var err error
+	for attempt.Next() {
+		err = w.validate(status)
+		if err == nil {
+			break
+		}
+		if attempt.More() {
+			logger.Debugf("validation failed (retrying): %v", err)
```

Comment:
Info perhaps?

Reply:
My thinking was that if you've got logging set to info and something eventually passes you probably don't want the preceding failed attempts logged.

Reply:
Fair enough.